### ISSUE TITLE
Verify release version matches build

### ISSUE THREAD

**Issue by antonagestam** [less-action/reusables#4]
The python-publish workflow should check that the tagged version matches the version of the built package, and fail otherwise.

**Comment by antonagestam** [comment]
This is currently broken, should use `pkginfo.Wheel`, e.g. something like:

```python
from pathlib import Path
from pkginfo import Wheel

dist = Path("dist")
wheel = Wheel(next(dist.glob("*.whl")))
print(wheel.version)
```

**Comment by antonagestam** [comment]
Fixed in #8
